extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -264,20 +264,46 @@ export class SchemaCompareResult {
 		return data;
 	}
 
+	private getFormattedScript(diffEntry: azdata.DiffEntry, getSourceScript: boolean): string {
```

Comment:
>getSourceScript [](start = 57, length = 15)

Why not pass in the script to format instead of using this boolean?

Reply:
all the children scripts of the diff entry also need to be formatted